modules/querying/pages/query-optimizer/enable-cost-optimizer.adoc

```diff
@@ -12,22 +12,13 @@ This page walks you through the steps to enable cost-based optimization.
 The following is the procedure to enable cost-based optimization when installing a query.
 
 === Identify referenced types and attributes
-You need to identify the vertex and edge types for which you need to compute the following statistics:
+You need to identify the vertex and edge attributes for which you need to compute histogram statistics, which refers to the distribution of values for an attribute.
 
-* Cardinality: the number of instances for each type
-* Histogram: the distribution of values for an attribute
-
-==== Cardinality
-You need to compute cardinality data for the following vertex and edge types:
-
-* Any vertex type referenced in the path pattern of a `FROM` clause.
-* Any _refined edge type_.
-A _refined edge type_ means an edge type with a specified pair of `from` and `to` vertex types as well as their reverse edge types if they exist.
```

Comment:
We haven't completely integrated automatic cardinality stats collection, so this API still exists.